ui/dashboardApp/index.ts

```diff
@@ -38,6 +39,7 @@ async function main(options: AppOptions) {
   )
 
   apiClient.init()
+  reportData.init()
```

Comment:
`telemetry.init`

Reply:
Need to await?

Reply:
Yep, you're right, else we need to return a "./config.js" in the html head.